test_runner/src/main/kotlin/ftl/args/CreateCommonArgs.kt

```diff
@@ -15,7 +15,7 @@ fun CommonConfig.createCommonArgs(
     // gcloud
     devices = gcloud::devices.require(),
     resultsBucket = ArgsHelper.createGcsBucket(
-        projectId = flank::project.require(),
```

Comment:
Why this change? Project id must never be empty

Reply:
reverted